temba/contacts/templatetags/contacts.py

```diff
@@ -199,7 +199,7 @@ def history_class(item):
     if isinstance(item, Msg):
         if item.media and item.media[:6] == 'video:':
             css = '%s %s' % (css, 'video')
-        if item.direction or item.recipient_count:
+        if item.direction or (item.broadcast and item.broadcast.recipient_count > 1):
```

Comment:
@rowanseymour thanks for the review but I'm not sure what you're asking me here. Could you elaborate a bit? I've copied the approach taken in the [`activity_icon()` template tag](https://github.com/praekeltfoundation/rapidpro/blob/406dc784bb40b799021bcd41386be37629a46ad3/temba/contacts/templatetags/contacts.py#L163).